src/Bundle/test/app/AppKernel.php

```diff
@@ -49,23 +40,9 @@ public function registerBundles()
         return $bundles;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function registerContainerConfiguration(LoaderInterface $loader)
+    public function registerContainerConfiguration(LoaderInterface $loader): void
     {
-        $loader->load(__DIR__ . '/config/config_' . $this->getEnvironment() . '.yml');
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function getContainerBaseClass()
-    {
-        if (0 === strpos($this->environment, 'test')) {
-            return MockerContainer::class;
-        }
-
-        return parent::getContainerBaseClass();
+        $loader->load(__DIR__ . '/config/{config}.{php,xml,yaml,yml}', 'glob');
+        $loader->load(__DIR__ . '/config/{config}_' . $this->getEnvironment() . '.{php,xml,yaml,yml}', 'glob');
```

Comment:
Makes the environment config file optional, it's the same behaviour as in Symfony 4+ apps.